uint8 MODE_VELOCITY=0
uint8 MODE_ABSOLUTE=1

uint8 mode
float64 target_distance   # meters (absolute position)
float64 linear_vel        # m/s